modules/msk/snppileup/main.nf

```diff
@@ -0,0 +1,56 @@
+process SNPPILEUP {
+    tag "$meta.id"
+    label 'process_medium'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://mskcc/facets-suite:2.0.9':
+        'docker.io/mskcc/facets-suite:2.0.9' }"
+
+    input:
+
+    tuple val(meta),  path(normal), path(normal_index)
+    tuple val(meta1), path(tumor),  path(tumor_index)
```

Comment:
seems like it might be useful to join the tumor and normal into the same tuple. i think the way you have it written, when processing multiple tumor and normal samples, all elements in the channel will be paired with all other elements in the channel. so if you have 5 tumors and 5 normals, you will have 25 tasks launched, which might not be ideal.

Reply:
Input channels have been updated, let me know if that works!

Reply:
looks good to me!